runtime/devnet/Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/paritytech/extended-parachain-template/"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [build-dependencies]
-substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", optional = true, branch = "polkadot-v0.9.43" }
+substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", optional = true, branch = "polkadot-v1.0.0" }
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
```

Comment:
Minor updates to `log`, `scale-info`, `smallvec`.

Reply:
Fixed in [1c74d00](https://github.com/paritytech/extended-parachain-template/pull/54/commits/1c74d008b375d31f5677f61f93e0986f9a817b20).